packages/components/src/components/hds/copy/snippet/index.hbs

```diff
@@ -7,7 +7,7 @@
   type="button"
   class={{this.classNames}}
   {{hds-clipboard text=@textToCopy onSuccess=this.onSuccess onError=this.onError}}
-  aria-label={{concat 'copy ' @textToCopy}}
+  aria-label={{concat "copy " @textToCopy}}
```

Comment:
was getting linter error so I went ahead and fixed

Reply:
we missed [linting components/src](https://github.com/hashicorp/design-system/pull/2003) since the v2 addon migration – these would be captured by CI moving forward

Reply:
Should I remove this change then?

Reply:
if you rebase your branch against `main` this change should just disappear

Reply:
Oh great, let me do that then. Thanks!